src/Core/src/Handlers/WebView/WebViewHandler.Android.cs

```diff
@@ -11,23 +11,22 @@ protected override AWebView CreateNativeView()
 		{
 			var aWebView = new MauiWebView(Context!)
 			{
-#pragma warning disable 618 // This can probably be replaced with LinearLayout(LayoutParams.MatchParent, LayoutParams.MatchParent); just need to test that theory
-				LayoutParameters = new AbsoluteLayout.LayoutParams(LayoutParams.MatchParent, LayoutParams.MatchParent, 0, 0)
-#pragma warning restore 618
+				LayoutParameters = new LayoutParams(LayoutParams.MatchParent, LayoutParams.WrapContent)
 			};
 
-			if (aWebView.Settings != null)
-			{
-				aWebView.Settings.JavaScriptEnabled = true;
-				aWebView.Settings.DomStorageEnabled = true;
-			}
 			return aWebView;
 		}
+		protected override void ConnectHandler(AWebView nativeView)
+		{
+			MapWebChromeClient(this, VirtualView);
```

Comment:
Does this look correct @PureWeen  @hartez  .. this was stuff that was normally on the CreateNativeView or in the ctor of the WebView

Reply:
Do you need to call these here? Since we have the mapper methods now those should get called at startup just fine yea?

Reply:
1) If order matters then we probably enforce that order on the handler somehow. So like have each 3 of these call something like "ProcessSourceIfReady" which checks to see if all necessary parts have been initialized? 

2) If we do end up still wanting to call `MapWebChromeClient` I think the way to do that is via `Handler.UpdateValue(nameof(WebChromeClient))` that way it'll call whatever the mapper is that's defined. If you just call ours then it's not going to run through the mapper if users replace the behavior

Reply:
Yes the n 2 i agree you are right.. why do you think n1 calling them on the ConnectHandler is bad ?

Reply:
Since we havea CreateNativeView, should we have a SetupNativeView ? where Mappers can be called ?

Reply:
> Yes the n 2 i agree you are right.. why do you think n1 calling them on the ConnectHandler is bad ?

I just don't think it accomplishes anything here other than adding a redundant call to those mappers. Is there a scenario where not having these here will make something impossible to do?

Reply:
It's reduddant i agree, but like setting the settings and the client needs to be before loading the source, thats why settings are set on ctor normally 

If you call MapSource, but didn't enable javascript for example it want work when you load

Reply:
@rmarinho can we check if they've ran? So each three of those things call some method
`DoThingsIfAllThingsAreReady()`

And once all three of those things have ran once then we call that method? Track it via a flag or bools or a counter for now?

Reply:
Which is fine because that's where our conversations on purpose has landed. State all goes inside the handler. It's basically just a State Bag that sits alongside the native/virtual controls. `DoThingsIfAllThingsAreReady` can sit inside the MauiWebView and be virtual

Reply:
ok pushed again @PureWeen